--- Library/Homebrew/brew.sh.orig	2025-09-26 08:01:50.000000000 +0900
+++ Library/Homebrew/brew.sh	2025-09-27 13:36:22.389730649 +0900
@@ -675,6 +675,13 @@
 
   curl_version_output="$(${HOMEBREW_CURL} --version 2>/dev/null)"
   curl_name_and_version="${curl_version_output%% (*}"
+
+  echo "ててて"
+  echo $curl_version_output
+  echo $curl_name_and_version
+  numeric "${curl_name_and_version##* }"
+  numeric "${HOMEBREW_MINIMUM_CURL_VERSION}"
+
   if [[ "$(numeric "${curl_name_and_version##* }")" -lt "$(numeric "${HOMEBREW_MINIMUM_CURL_VERSION}")" ]]
   then
     message="Please update your system curl or set HOMEBREW_CURL_PATH to a newer version.
